build.gradle

```diff
@@ -60,3 +61,27 @@ jacocoTestReport {
 }
 
 build.dependsOn jacocoTestReport
+
+pmd {
+    toolVersion = '5.4.1'
+    ignoreFailures = false
+    ruleSetFiles = files('./code_quality_tools/pmd.xml')
+}
+
+task('pmd', type: Pmd, dependsOn: 'assemble') {
+    description = 'Run pmd'
+    group = 'verification'
+
+    ruleSets = []
+
+    source = fileTree('src/main/java')
+    include '**/*.java'
+    exclude '**/gen/**'
+
+    reports {
+        html.enabled = true
+        xml.enabled = true
+    }
+}
+
+build.dependsOn 'pmd'
```

Comment:
Yeah I did the change through Sublime. I have it activated on my AS + IntelliJ. 👍